src/App/App.js

```diff
@@ -77,6 +84,7 @@ render() {
     return (
         <>
         <Nav/>
+        <span className="error">{this.state.errorMessage}</span>
```

Comment:
This looks good so far, we might have to refactor it when we start looking at routing.